configuration.go

```diff
@@ -270,6 +272,12 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
 	defaultDocker.ExposedByDefault = true
 	defaultDocker.Endpoint = "unix:///var/run/docker.sock"
 
+	//default Swarm
+	var defaultSwarm provider.Swarm
+	defaultSwarm.Watch = true
+	defaultSwarm.ExposedByDefault = true
+	defaultSwarm.Endpoint = "unix:///var/run/docker.sock"
```

Comment:
@diegofernandes I recently built an image using your master branch and tried the following in my docker swarm cluster.

```
docker service create --replicas 3 --publish 80:80 --mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock --name traefik-swarm timchunght/traefik:swarm --web --docker --docker.domain=docker.localhost --logLevel=DEBUG

docker service create --replicas 3 --name whoami --label "traefik.frontend.rule=Host:whoami.docker.localhost" emilevauge/whoami 

curl -H Host:whoami.docker.localhost http://127.0.0.1
```

but it doesn't seem to work yet. the `curl` command returned `404 page not found`, which means that the traefik container seems to be running perfectly. I tried the same traefik container on a single node docker server using the docker-compose config at https://docs.traefik.io/#docker and it worked. Let me know and I will be happy to test the new solutions. Thanks